### PR TITLE
updates arvo-vs-azimuth.udon

### PR DESCRIPTION
**Pull request by rmariani**
This is more or less a complete rewrite. I took it out of the weeds and made it clearer. @galenwp, I am curious about your opinion on this new copy.

### CODE REVIEW

File: introduction/arvo-vs-azimuth.udon [hunk 1]
```diff
@@ -4,12 +4,12 @@
...
+*Arvo* is an operating system that provides the software for a personal server. These personal servers together constitute the peer-to-peer Arvo network. To make this network work on the social level, Arvo is built to work with a system of scarce and immutable identities.
 
-However, being a decentralized system, we need some way to cryptographically secure and authenticate your identity on the Arvo network. What keeps track of who owns any given identity? Well, simply matching something to an address is a small piece of computing, and a blockchain are wonderful for such a purpose. That's where the Azimuth comes in.
+*Azimuth* is the public-key infrastructure built to be such a system. A suite of smart-contracts on the Ethereum blockchain, Azimuth determines which Ethereum addresses own which Azimuth identities, called _points_. All point-related operations, such as transfers, are governed at this layer. But Azimuth isn't necessarily Arvo-related -- it could be used as a generalized identity system for other projects.
```
Comment (Fang-) [inline introduction/arvo-vs-azimuth.udon]: "But Azimuth isn't necessarily Arvo-related" might be better worded as "But Azimuth isn't built strictly for Arvo" or similar. Your discretion though.

**Comment by rmariani** [comment]
That's a good idea

**Comment by Fang-** [comment]
Also "could" vs "can", but maybe that's getting a bit nitpicky. +1 on this PR overall! Very important section. (:

**Comment by rmariani** [comment]
I think "could" is better here because it's less distancing from Urbit and implies that other uses are potential. Nothing else uses it!

**Comment by Fang-** [comment]
> Nothing else uses it!

_shakes fist_, just you wait!

**Comment by msutherl** [comment]
@rmariani @baudtack I caught a typo and also have some quibbles about wording – where's the best place to leave copy edits?

**Comment by msutherl** [comment]
hah – too late, I'll batch some copy edits with another PR another time

**Comment by rmariani** [comment]
Just in a PR I think would be best. But yeah if you have many copy edits across the board, those could all be on one branch

**Pull request merged**